### DISCUSSION TITLE
Selected items aren't as distinct in discussion view

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#108148] [Mobile] [answered]
GitHub Mobile for Android beta 
(This is a Google Pixel 6A, but I have the same problem on my Pixel 3A.)
I find it much easier to see that I selected this ⬆️:

![Screenshot_20240213-083747.png](https://github.com/community/community/assets/2119212/12baf089-f5e3-42a8-a33e-113233001b5c)

Than that I selected this ⬆️:

![Screenshot_20240213-083815.png](https://github.com/community/community/assets/2119212/ea9e0d59-5258-455e-9178-4eed3b4d7f5d)

I'm constantly unintentionally removing my ⬆️👍 from items because the "you've selected" this indication isn't strong enough.

**Comment by HenSquared** [comment, marked as answer]
Thanks @jsoref ,

My understanding of the issue you're experiencing is:
The design of the selected emoji reaction button is not distinct enough. When you are trying to select an emoji reaction, you accidentally select an already selected emoji reaction, because the selected state is not clear enough. 

Let me know if I got that right! In the meantime, I will open an issue for this.

**Reply by jsoref** [reply]
Yep, and when I select an already selected emoji/vote thing, that undoes the reaction/vote thing.

**Comment by jsoref** [comment]
I wonder if my problem stems from posts where I visit them in response to a notification... 
![Screenshot_20240220-091214.png](https://github.com/community/community/assets/2119212/d91c983f-bff3-416c-a604-893a924363cd)

This notification might make no sense as it looks like I shouldn't have been able to 👍 it yet as it's for a new message. But the notification came to many devices, so on one device at some point I 👍d it and then much later I saw the notification for the same event.
